addons/crm/security/ir.model.access.csv

```diff
@@ -24,3 +24,5 @@ access_crm_lead_lost,access.crm.lead.lost,model_crm_lead_lost,sales_team.group_s
 access_crm_lead2opportunity_partner,access.crm.lead2opportunity.partner,model_crm_lead2opportunity_partner,sales_team.group_sale_salesman,1,1,1,0
 access_crm_lead2opportunity_partner_mass,access.crm.lead2opportunity.partner.mass,model_crm_lead2opportunity_partner_mass,sales_team.group_sale_salesman,1,1,1,0
 access_crm_merge_opportunity,access.crm.merge.opportunity,model_crm_merge_opportunity,sales_team.group_sale_salesman,1,1,1,0
+access_crm_recurring_plan_manager,crm.recurring.plan.manager,model_crm_recurring_plan,sales_team.group_sale_manager,1,1,1,1

```

Comment:
Side-guidelining-notes: would be better with something like crm_recurring_plan_access_manager,crm.recurring.plan.access.manager, ... (model, type (acces rule), sub-group) 

Tde, guidelines-man.